pkg/sfu/downtrack.go

```diff
@@ -398,6 +401,13 @@ func (d *DownTrack) Unbind(_ webrtc.TrackLocalContext) error {
 	return nil
 }
 
+func (d *DownTrack) SetTrailer(trailer []byte) {
```

Comment:
if this could be set during downtrack construction, we could skip this function and locks?

Reply:
Once again, I had that in the first version :-). The down track constructor is getting to be longish. Maybe, I should just make a `DownTrackParams`. Will add it as an argument for now.